.ddev/commands/web/openmage-install

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+## ProjectTypes: magento
+## Description: Install OpenMage
+## Usage: openmage-install [-d|--default] [-s|--sampledata] [-k|--keep] [-q|--quiet]
+## Example: ddev openmage-install -d -s -k
+## Flags: [{"Name":"default","Shorthand":"d","Usage":"use default values"},{"Name":"sampledata","Shorthand":"s","Usage":"install sample data"},{"Name":"keep","Shorthand":"k","Usage":"keep sample data package"},{"Name":"quiet","Shorthand":"q","Usage":"silent install with sample data"}]
+
+ROOT="${PWD}"
+
+QUIET_INSTALL_FLAG=''
+SAMPLE_DATA_FLAG=''
+SAMPLE_DATA_KEEP_FLAG=''
+USE_DEFAULT_FLAG=''
+
+while :; do
+     case ${1:-} in
+         -d|--default)
+            USE_DEFAULT_FLAG='true' ;;
+         -s|--sampledata)
+            SAMPLE_DATA_FLAG='true' ;;
+         -k|--keep)
+            SAMPLE_DATA_KEEP_FLAG='true' ;;
+         -q|--quiet)
+            QUIET_INSTALL_FLAG='true'
+            USE_DEFAULT_FLAG='true'
+            SAMPLE_DATA_FLAG='true'
+            ;;
+         --)              # End of all options.
+             shift
+             break
+             ;;
+         -?*)
+             printf 'WARN: Unknown option (ignored): %s\n' "$1" >&2
+             ;;
+         *)               # Default case: No more options, so break out of the loop.
+             break
+     esac
+
+     shift
+ done
+
+LOCALXML="${ROOT}/app/etc/local.xml"
+if [ -f "${LOCALXML}" ]; then
+  if [[ "${QUIET_INSTALL_FLAG}" ]]; then
+    DELETE='y'
+  else
+    read -r -p "OpenMage is already installed. Delete local.xml and drop database [y/N] " DELETE
+    DELETE=${DELETE,,} # to lower
+  fi
+
+  if [[ "${DELETE}" =~ ^(yes|y) ]]; then
+    mysql -u db -h db -e "DROP SCHEMA IF EXISTS db; CREATE SCHEMA db;";
+    rm "${LOCALXML}"
+  else
+    exit 1
+  fi
+fi
+
+# install sample data
+if [[ "${SAMPLE_DATA_FLAG}" ]]; then
+  INSTALL_SAMPLE_DATA='y'
+else
+  read -r -p "Install sample data? [y/N] " INSTALL_SAMPLE_DATA
+  INSTALL_SAMPLE_DATA=${INSTALL_SAMPLE_DATA,,} # to lower
+fi
+
+if [[ $INSTALL_SAMPLE_DATA =~ ^(yes|y) ]]; then
+  SAMPLE_DATA_URL=https://github.com/Vinai/compressed-magento-sample-data/raw/master/compressed-magento-sample-data-1.9.2.4.tgz
+  SAMPLE_DATA_DIRECTORY="${ROOT}/.sampleData"
+  SAMPLE_DATA_FILE=sample_data.tgz
+
+  if [[ ! -d "${SAMPLE_DATA_DIRECTORY}" ]]; then
+    echo "Creating backup directory ..."
+    mkdir -p "${SAMPLE_DATA_DIRECTORY}"
+  fi
+
+  cd "${SAMPLE_DATA_DIRECTORY}" || exit
+  if [[ ! -f "${SAMPLE_DATA_DIRECTORY}/${SAMPLE_DATA_FILE}" ]]; then
+    echo "Downloading sample data ..."
+    wget "${SAMPLE_DATA_URL}" -O "${SAMPLE_DATA_FILE}"
+  fi
+
+  echo "Uncompressing sample data ..."
+  tar xf "${SAMPLE_DATA_FILE}"
+
+  echo "Copying sample data ..."
+  cp -r magento-sample-data-1.9.2.4/* "${PWD}/"
```

Comment:
There is an issue in this command. Its execution is done in the .sampleData directory and thus both /media and /skin are copied here and not in the root directory of OpenMage as it should be.

Here is the fix

```cp -r magento-sample-data-1.9.2.4/* "${ROOT}/"```